api/s2n.h

```diff
@@ -66,6 +66,15 @@ typedef enum {
     S2N_EXTENSION_CERTIFICATE_TRANSPARENCY = 18,
 } s2n_tls_extension_type;
 
+typedef enum {
+    S2N_TLS_MAX_FRAG_LEN_EXT_NONE,     /* don't use the extension */
+    S2N_TLS_MAX_FRAG_LEN_512,
+    S2N_TLS_MAX_FRAG_LEN_1024,
+    S2N_TLS_MAX_FRAG_LEN_2048,
+    S2N_TLS_MAX_FRAG_LEN_4096,
+    S2N_TLS_MAX_FRAG_LEN_INVALID,      /* first invalid mfl_code value */
```

Comment:
Yea definitely possible. similar thinking process to S2N_TLS_MAX_FRAG_LEN_EXT_NONE, wanted to explicitly show that enum values starting from 5 and up are invalid. (used in invalid value test cases, and checks in s2n_client_extensions & s2n_config. 

I will take out both if these two are unnecessary